src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/SVClusterIntegrationTest.java

```diff
@@ -533,5 +540,34 @@ public void testAllosome() {
         }
         Assert.assertEquals(expectedRecordsFound, 1);
     }
+    @Test
+    public void testCleanedVcf() {
+        final File output = createTempFile("cleaned_vcf_cluster", ".vcf");
+        // Note we use very loose clustering criteria on a normal cleaned vcf to ensure some clustering happens
+        final ArgumentsBuilder args = new ArgumentsBuilder()
+                .addOutput(output)
+                .addVCF(getToolTestDataDir() + "bwa_melt.cleaned.chr22_chrY.vcf.gz")
+                .add(SVCluster.PLOIDY_TABLE_LONG_NAME, getToolTestDataDir() + "1kgp.batch1.ploidy.tsv")
+                .add(SVCluster.VARIANT_PREFIX_LONG_NAME, "SVx")
+                .add(SVCluster.ALGORITHM_LONG_NAME, SVCluster.CLUSTER_ALGORITHM.SINGLE_LINKAGE)
+                .add(StandardArgumentDefinitions.REFERENCE_LONG_NAME, REFERENCE_PATH)
+                .add(SVClusterEngineArgumentsCollection.DEPTH_SAMPLE_OVERLAP_FRACTION_NAME, 0)
+                .add(SVClusterEngineArgumentsCollection.DEPTH_INTERVAL_OVERLAP_FRACTION_NAME, 0.1)
+                .add(SVClusterEngineArgumentsCollection.DEPTH_BREAKEND_WINDOW_NAME, 10000000)
+                .add(SVClusterEngineArgumentsCollection.MIXED_SAMPLE_OVERLAP_FRACTION_NAME, 0)
+                .add(SVClusterEngineArgumentsCollection.MIXED_INTERVAL_OVERLAP_FRACTION_NAME, 0.1)
+                .add(SVClusterEngineArgumentsCollection.MIXED_BREAKEND_WINDOW_NAME, 5000)
+                .add(SVClusterEngineArgumentsCollection.PESR_SAMPLE_OVERLAP_FRACTION_NAME, 0)
+                .add(SVClusterEngineArgumentsCollection.PESR_INTERVAL_OVERLAP_FRACTION_NAME, 0.1)
+                .add(SVClusterEngineArgumentsCollection.PESR_BREAKEND_WINDOW_NAME, 5000);
+
+        runCommandLine(args, SVCluster.class.getSimpleName());
+
+        final Pair<VCFHeader, List<VariantContext>> vcf = VariantContextTestUtils.readEntireVCFIntoMemory(output.getAbsolutePath());
+        final VCFHeader header = vcf.getKey();
+        Assert.assertEquals(header.getSampleNamesInOrder().size(), 161);
+        final List<VariantContext> records = vcf.getValue();
+        Assert.assertEquals(records.size(), 1227);
```

Comment:
This is sort of a "test smell" to me, but the clustering should be pretty resilient, right?

Reply:
Yes very